manifests/plugin/abrt.pp

```diff
@@ -0,0 +1,57 @@
+# = Foreman Proxy Abrt plugin
+#
+# This class installs abrt plugin
+#
+# === Parameters:
+#
+# $group::                      group owner of the configuration file
+#
+# $enabled::                    enables/disables the plugin
+#
+# $abrt_send_log_file::         Log file for the forwarding script.
+#
+# $spooldir::                   Directory where uReports are stored before they are sent
+#
+# $aggregate_reports::          Merge duplicate reports before sending
+#
+# $send_period::                Period (in seconds) after which collected reports are forwarded.
+#                               Meaningful only if smart-proxy-abrt-send is run as a daemon (not from cron).
+#
+# $faf_server_url::             FAF server instance the reports will be forwarded to
+#
+# $faf_server_ssl_noverify::    Set to true if FAF server uses self-signed certificate
+#
+#                               Following two options enable client authentication to FAF server:
```

Comment:
kafo/foreman-installer won't understand this, you'll need to put the description after both parameters